bin/oneoffs/migrate_storage.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
+import argparse
+import datetime
+import logging
+import os
+import uuid
+
+import fs.path
+import fs.move
+
+from api import config, util
+
+log = logging.getLogger('migrate_storage')
+log.setLevel(logging.INFO)
+
+COLLECTIONS_PREFIXES = [('projects', 'files'),
+                            ('acquisitions', 'files'),
+                            ('analyses', 'files'),
+                            ('sessions', 'files'),
+                            ('sessions', 'subject.files'),
```

Comment:
I don't think we have any actual users using subject files to be clear, but it is good to include them just in case. It is an undocumented feature of project upload.